hydra/core/config_store.py

```diff
@@ -133,10 +133,10 @@ def get_type(self, path: str) -> ObjectType:
     def list(self, path: str) -> List[str]:
         d = self._open(path)
         if d is None:
-            raise IOError(f"Path not found {path}")
+            raise OSError(f"Path not found {path}")
 
         if not isinstance(d, dict):
-            raise IOError(f"Path points to a file : {path}")
+            raise OSError(f"Path points to a file : {path}")
```

Comment:
Why is this change necessary?

Reply:
It's not strictly necessary because they are semantically the same, but it's the recommended canonical name now. See https://stackoverflow.com/a/29347946/1165181

Reply:
```python
>>> IOError is OSError
True
```
I see. Thanks.